### PR TITLE
LPS-84241 Show label on Calculator output selection field

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz** [comment]
Hi @alinedoleron , I sent `LPS-84231` to BChan here. We still need to work on `LPS-84305`. 

Please, rebase this before resending, thanks! :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_options_toolbar.js [hunk 1]
```diff
@@ -107,18 +107,27 @@ AUI.add(
...
 						var dropdown = target.ancestor('.dropdown-menu');
 
-						var handlerName = target.getData('handler');
-
 						var formBuilder = instance.get('formBuilder');
 
-						var field = instance.get('field');
+						var field = '';
+
+						if (dropdown.attr('aria-labelledby') == 'dropdownFieldToolbar') {
```
Comment (pedroqueiroz) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_options_toolbar.js]: Hey @alinedoleron , can we be more specific in this condition?

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by pedroqueiroz** [comment]
Second part resent at https://github.com/rafaprax/liferay-portal/pull/825.

**Pull request closed without merging**